sdk/eventhubs/azure-messaging-eventhubs/test/ut/consumer_client_test.cpp

```diff
@@ -311,6 +311,9 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace Test {
     {
       GTEST_SKIP();
     }
+
+    // Disabled test for now.
```

Comment:
If intending to keep this test as disabled and merging, consider renaming the test with a `DISABLED_` prefix instead of using `#if 0`

`DISABLED_RetrieveMultipleEvents` should work as expected.